app/src/main/java/com/example/workoutapp/RecordActivity.kt

```diff
@@ -5,10 +5,10 @@ import android.widget.Toast
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.compose.foundation.layout.*
-import androidx.compose.material.Button
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
+import androidx.compose.material.*
```

Comment:
I'd leave this as it were. Wildcard imports (*) are considered bad practice for a couple of reasons:

- Less documentation on what classes are being imported.
- If you have two wildcard imports, and they have a class with the same name, there will be a collision. Example: java.util.* and java.sql.* both have a class named Date. Different classes, same name. So, if you're using wildcards, which Date class should be used?
- It makes no difference in runtime performance, as the compiler will automatically optimize imports.